### PR TITLE
Add RequireJS example

### PR DESCRIPTION
**Pull request by pbernasconi**

### CODE REVIEW

File: examples/requirejs/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Then load up <http://localhost:8000>!
+
+And that's it!
+
+[1]: http://expressjs.com
+[2]: https://github.com/plaid/plaid-node
+[3]: https://github.com/plaid/link
+[4]: https://plaid.com/account/
+[5]: https://plaid.com/docs#sandbox
```
Comment (michaelckelly) [inline examples/requirejs/README.md]: This should now be https://plaid.com/docs/api#sandbox - I'll update the other READMEs.

**Comment by michaelckelly** [comment]
🐢

**Comment by jjperezaguinaga** [comment]
@pbernasconi Looks good! Let me give a quick try locally :)

**Comment by jjperezaguinaga** [comment]
@pbernasconi Other than the changes we discussed offline, you got an 🐺

**Pull request merged**